.github/workflows/ci.yml

```diff
@@ -1,79 +1,72 @@
 name: ci
 
 on:
-  push:
-    branches:
-      - '**'
-    tags:
-      - 'v*'
-  pull_request:
-    branches:
-      - 'master'
+    push:
+        branches:
+            - "**"
+        tags:
+            - "v*"
+    pull_request:
+        branches:
+            - "master"
+    workflow_dispatch:
```

Comment:
what is workflow dispatch?

Reply:
When I was trying to figure out why workflows weren't running for those other PRs, I looked up how to be able to trigger them manually.

This allows you to (from actions if its on `master`) or via gh workflow run, to manually trigger workflows to run.

Reply:
ah, I see.